ThreeMigrationGuide.md

```diff
@@ -1,3 +1,24 @@
-### Texture 3.0 Migration Guide ###
+## Texture 3.0 Migration Guide ###
```

Comment:
The two sides are now imbalanced.

Reply:
Why is there trailing pounds? The trailing is invisible, only the leading is required for the header formatting

Reply:
I'm more inclined to remove the trailing ones than making it match the leading ones

Reply:
I agree. Let's remove the trailing characters.

Reply:
```suggestion
## Texture 3.0 Migration Guide
```